### PR TITLE
button and placeholder edits

### PR DESCRIPTION
**Pull request by elywelly**
- Edited login button to be consistent to sign up hover
- Added default user event creator as user 1 
- Fixed all error handling
- Error to be displayed at the top of section/forms (error in red and success in green - for update status)

### CODE REVIEW

File: controllers/sessions/index.js [hunk 1]
```diff
@@ -47,6 +47,7 @@ router.get('/', (req, res) => {
             user_id: req.session.user_id,
             email: req.session.email,
             first_name: req.session.first_name,
+            username: req.session.username,
```
Comment (wprk) [inline controllers/sessions/index.js:50]: Do you need to store all this data in your session? I'm not sure you'd want much more than the user_id stored here.

Comment (elywelly) [reply controllers/sessions/index.js:50]: We are pulling these data to use throughout the site - e.g. in this PR to auto populate the logged in user as User 1.

Comment (kdaya001) [reply controllers/sessions/index.js:50]: I also don't think we actually use first name anywhere do we? We can probably remove that line, same with email other than when we validate?

Comment (elywelly) [reply controllers/sessions/index.js:50]: Yeap I think you're right, I looked through I don't think we are using those, mostly user ID and username, added a trello card to remove.

**Review by kdaya001 — APPROVED** [review]

**Review by May-Manmai — APPROVED** [review]

**Pull request merged**